src/reducers/monitors.js

```diff
@@ -0,0 +1,27 @@
+const UPDATE_MONITORS = 'scratch-gui/monitors/UPDATE_MONITORS';
+
+const initialState = {
+    monitors: []
+};
+
+const reducer = function (state, action) {
+    if (typeof state === 'undefined') state = initialState;
+    switch (action.type) {
+    case UPDATE_MONITORS:
+        return Object.assign({}, state, {monitors: action.monitors});
```

Comment:
If we want this to be less painful than the way targets work, then I think `UPDATE_MONITORS` should always supply the entire list of monitors with all the details. Then this can just return `action.monitors` rather than mucking around with merging (which can get out of sync when projects are loaded on top of each other).

Alternatively we should have `ADD_MONITOR`, `REMOVE_MONITOR`, `UPDATE_MONITOR` instead of dealing with the list as a whole.  But I think doing the whole list for now should be sufficient.